### ISSUE TITLE
Could you add copyRealmIfMigrationNeeded()?

### ISSUE THREAD

**Issue by grub-** [realm/realm-java#1817]
It would be really good to have method `copyRealmIfMigrationNeeded()` because of constant breacking changes.

**Comment by cmelchior** [comment]
What should such a method do? 

We only introduced breaking changes to the Realm schema in 0.82 and 0.83. Both of these required changes to your model files, which could not be automatically migrated by copying the file.

That said, I am sorry you feel that we are constantly breaking things. We try to be mindful about it, but at the same time we do want to have the best API possible before reaching 1.0.

**Comment by grub-** [comment]
Yes but if model of app simple and don't need migration. It would be easier just to copy data to new db. Now I'm using `deleteRealmIfMigrationNeeded()` and forcing redownload data from server.

**Comment by jjhesk** [comment]
it seems to be crashing to with this code
```java
   return new RealmConfiguration.Builder(app)
                .deleteRealmIfMigrationNeeded()
                .build();
```

**Comment by beeender** [comment]
@jjhesk What is the crash message? If it is not related with this issue, would you please create a new issue for that?
